js/src/forum/components/NotificationGrid.js

```diff
@@ -133,7 +123,7 @@ export default class NotificationGrid extends Component {
    * @param {String} method
    */
   toggleMethod(method) {
-    const keys = this.types.map((type) => this.preferenceKey(type.name, method)).filter((key) => !this.inputs[key].props.disabled);
+    const keys = this.types.map((type) => this.preferenceKey(type.name, method)).filter((key) => key in this.props.user.preferences());
```

Comment:
I'm not sure if babel correctly transpiles this for older browsers.

Then again, I don't know what browsers we want to support anymore 🤔

Reply:
As per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in, looks like it's supported widely?

Reply:
Oh. I never use it and assumed it to be newer syntax 🤔. Ignore me.